frontend/apps/reader/modules/readercropping.lua

```diff
@@ -111,20 +53,59 @@ function ReaderCropping:onPageCrop(mode)
     else
         self.ui:handleEvent(Event:new("SetZoomMode", "page", "cropping"))
     end
-    self.ui:handleEvent(Event:new("SetDimensions",
-        Geom:new{w = Screen:getWidth(), h = math.floor(Screen:getHeight()*11/12)})
-    )
+
+    -- prepare bottom buttons so we know the size available for the page above it
+    local button_table = ButtonTable:new{
+        width = Screen:getWidth(),
+        button_font_face = "cfont",
+        button_font_size = 20,
+        buttons = {{
+            {
+                text = _("Cancel"),
+                callback = function() self:onCancelPageCrop() end,
+            },
+            {
+                text = _("OK"),
```

Comment:
In line with our more recent UI guidelines we could also change it to something like `Crop`, `Apply (page) crop`, `Crop to … selection?`.

Reply:
Let's go with `Cancel | Apply crop` to keep it short.